README.md

```
@@ -1,67 +1,81 @@
 [![Coverage Status](https://coveralls.io/repos/github/vbpf/ebpf-verifier/badge.svg?branch=main)](https://coveralls.io/github/vbpf/ebpf-verifier?branch=main)[![CodeQL](https://github.com/vbpf/ebpf-verifier/actions/workflows/codeql-analysis.yml/badge.svg?branch=main)](https://github.com/vbpf/ebpf-verifier/actions/workflows/codeql-analysis.yml)
 
-# PREVAIL
+# PREVAIL - A new eBPF verifier
 ## a Polynomial-Runtime EBPF Verifier using an Abstract Interpretation Layer
 
-A new eBPF verifier.
+
```

Markdownlint

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)